src/App.js

```diff
@@ -1,25 +1,28 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/react-in-jsx-scope */
```

Comment:
For React 17, you need to add a rule in `.eslintrc.js` .. you need not add this rule in every file.

pls refer .. https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md

Reply:
added